### DISCUSSION TITLE
How to implement EarlyStopping in MONAI Label server

### DISCUSSION THREAD

**Discussion by amitjc** [Project-MONAI/MONAILabel#1723] [Q&A] [answered]
Dear experts,

Seek your guidance to implement EarlyStopping in MONAI Label server for training segmentation from scratch.

Look forward to your inputs.

Thank you.

Best Wishes,
Amit.

**Comment by diazandr3s** [comment, marked as answer]
Hi @amitjc,

By default MONAI Label uses Early Stopping. You can check this class defined in the basic train class here: https://github.com/Project-MONAI/MONAILabel/blob/159e1425e16c937a11d568aa369c0772b388c8cb/monailabel/tasks/train/basic_train.py#L594C33-L598

You could define the patience argument (**early_stop_patience**) in the train configuration from the GUI: https://github.com/Project-MONAI/MONAILabel/blob/159e1425e16c937a11d568aa369c0772b388c8cb/monailabel/tasks/train/basic_train.py#L589C35-L589C42

Hope this helps,

**Comment by amitjc** [comment]
Thanks @diazandr3s.

I shall use the slicer plugin and modify the early_stop_patience.
